docs/reference-system.md

```diff
@@ -0,0 +1,303 @@
+
+- the parametric (or reduced) latitude $\beta$, which is the result of stretching the semi-minor axis $b$ to the length of the semi-major axis $a$, resulting in a sphere. This stretching displaces $P_0$ to $P_\beta$. The conversion is
+  ```{math}
+  \tan{\beta} = \frac{b}{a}\tan{\phi} = (1 - f)\tan{\phi} = \sqrt{1 - e^2}\tan{\phi}
+  ```
+- the geocentric latitude $\theta$, which is the spherical latitude for the local radius at $P_0$ (the distance between the center and $P_0$). The conversion is
+  ```{math}
+  \tan{\theta} = \frac{b^2}{a^2} \tan{\phi} = (1 - e^2)\tan{\phi}
+  ```
+- the authalic latitude $\xi$, which is the result of stretching $b$ and compressing $a$ such that the resulting sphere (the "authalic sphere") has the same surface area as the ellipsoid. In the process, $P_0$ is displaced to $P_\xi$ to keep the surface area of faces on the authalic sphere the same as on the ellipsoid.
+
+  The conversion is
+
+  ```{math}
+  q(\phi) &= \frac{(1-e^2)\sin{\phi}}{1 - e^2\sin^2{\phi}} - \frac{1-e^2}{2e} \ln{\left(\frac{1-e\sin{\phi}}{1 + e\sin{\phi}}\right)} \\
+  \sin{\xi} &= \frac{q(\phi)}{q(\frac{\pi}{2})}
+  ```
+
+  And the inverse has to be computed iteratively or through a series expansion.
+
+Note that $\beta$ and $\xi$ are latitudes for auxiliary spheres.
+
+```{jupyter-execute}
+---
+hide-code: true
+---
+from rich.table import Table
+import rich.jupyter
+from rich.console import Console
+
+a = 6378137
+f_ = 298.257223563
+f = 1 / f_
+e = np.sqrt(2 * f - f**2)
+
+b = a * (1 - f)
+
+geographic_latitude = 45.0
+parametric_latitude = float(np.rad2deg(np.arctan(b / a * np.tan(np.deg2rad(geographic_latitude)))))
+geocentric_latitude = float(np.rad2deg(np.arctan(b**2 / a**2 * np.tan(np.deg2rad(geographic_latitude)))))
+authalic_latitude = float(np.rad2deg(convert(np.deg2rad(geographic_latitude), e)))
+
+table = Table(caption=f"ellipsoidal latitudes for WGS84\n(a = {a:01f}, f_inv = {f:03f})")
+table.add_column("Latitude")
+table.add_column("Value", justify="right")
+table.add_column("x - ϕ", justify="right")
+table.add_row(r"Geographic (ϕ)", f"{geographic_latitude:.8f}", f"{geographic_latitude - geographic_latitude:.8f}")
+table.add_row(r"Parametric (β)", f"{parametric_latitude:.8f}", f"{parametric_latitude - geographic_latitude:.8f}")
+table.add_row(r"Geocentric (θ)", f"{geocentric_latitude:.8f}", f"{geocentric_latitude - geographic_latitude:.8f}")
+table.add_row(r"Authalic (ξ)", f"{authalic_latitude:.8f}", f"{authalic_latitude - geographic_latitude:.8f}")
+
+# rich.jupyter.print(table)
+console = Console()
+segments = console.render(table)
+html = rich.jupyter._render_segments(segments)
+text = console._render_buffer(segments)
+rich.jupyter.JupyterRenderable(html, text)
+```
+
+Karney, 2023[^2], describes conversion formulas that reduce the numerical round-off errors.
+
+[^1]: https://doi.org/10.3133/pp1395
+
+[^2]: https://doi.org/10.48550/arXiv.2212.05818
```

Comment:
Change doi to full paper: https://doi.org/10.1080/00396265.2023.2217604

aka: https://www.tandfonline.com/doi/full/10.1080/00396265.2023.2217604